orbit/Containerfile

```diff
@@ -5,8 +5,8 @@ RUN apk update && apk upgrade && apk add \
 	envsubst \
 	;
 
-COPY . /orbit
-WORKDIR /orbit
+COPY . /usr/local/src/orbit
```

Comment:
fair, I don't like /orbit, but I don't actually particularly care where it goes

Reply:
how about `/usr/local/share/` https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s11.html
> The /usr/share hierarchy is for all read-only architecture independent data files.

using the local version since the code isn't from a system package.